wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -5,9 +5,14 @@ use parking_lot::Mutex;
 
 use std::{ffi::CStr, mem, ptr, sync::Arc};
 
+/// Update After Bind is used on intel only, and all intel devices have at least 524288
+const MAX_UPDATE_AFTER_BIND_DESCRIPTORS_IN_ALL_POOLS: u32 = 524288;
```

Comment:
so if we make it non-Intel, we should grab this number from the adapter instead of hard-coding it here

Reply:
This is actually the lowest non zero value for _all_ vulkan devices.

Reply:
sure! Still, if we can grab an actual value from a physical device, no need to hard-code the constant in